test/go/race_conditions_integration_test.go

```diff
@@ -25,6 +26,7 @@ func annotate(queries []string, marginalia string) []string {
 }
 
 func TestSelectUpdateBinlogCopy(t *testing.T) {
+	t.Skip("Skipping for MySQL8")
```

Comment:
This is a pretty important test as it covers one of the Ghostferry race conditions that it is designed to handle. Although I think this is already replicated in the Ruby test suite?

Reply:
We've fixed this test, so no need to skip it any more.